schema-to-pojo-core/src/main/java/org/sagebionetworks/schema/generator/handler/schema03/JSONMarshalingHandlerImpl03.java

```diff
@@ -106,9 +105,10 @@ protected JMethod createConstructor(ObjectSchema classSchema, JDefinedClass clas
         	JInvocation invocation = JExpr.invoke("super");
         	body.add(invocation);
         }
+        JFieldRef staticMessageRef = classType.owner().ref(ObjectSchema.class).staticRef("OBJECT_ADAPTER_CANNOT_BE_NULL");

```

Comment:
Concrete type is a constant in classes that implement interfaces.  Should it be there?